.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Bug Report
```

Comment:
kind: bug-report?

kind: feature-request?

kind: question?

Reply:
@rohitsardesai83 i have added labels for the same

Reply:
I have added the kind labels for bugs , enhancements , pull requests etc.

Reply:
I would suggest change `name` to `kind`. Thanks!